src/quo/components/list_items/account_list_card/schema.cljs

```diff
@@ -1,19 +1,19 @@
 (ns quo.components.list-items.account-list-card.schema)
 
 (def ^:private ?base
-  [:map
-   [:action {:optional true} [:enum :icon :none]]
-   [:blur? {:optional true} [:maybe :boolean]]
-   [:on-press {:optional true} [:maybe fn?]]
-   [:account-props
+  [:schema.common/map {:optional true}
+   [:action [:enum :icon :none]]
+   [:blur? [:maybe :boolean]]
+   [:on-press [:maybe fn?]]
+   [:account-props {:no-optional true}
```

Comment:
How to negate the parent `:optional` or `:maybe` property? By using `:no-optional` or `:no-maybe`. I chose these names because it's impossible to confuse them with each other. For example, if I used `:required` as the inverse of `:optional`, some people could be confused that `:required` is the inverse of `:maybe`.

Reply:
Side question 💭

Is it possible to merge two  `:schema.common/map`? For example, would I be able to define a map of required fields in one map and another map of optional or maybe fields and then merge the definitions together into a single map?